cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/workflow/WorkflowStatus.java

```diff
@@ -25,12 +25,12 @@
 
   private  Service.State state;
   private  WorkflowActionSpecification currentAction;
```

Comment:
Since the endpoint response has changed anyway (from string to map?), why not change the response to have `List<WorkflowNode>`?